### PR TITLE
Any instead of Count in IIS UrlRewriteFileParser

### PR DESCRIPTION
**Pull request by cnblogs-dudu**
Resolve the conversation in PR #11533.

### CODE REVIEW

File: src/Middleware/Rewrite/src/Internal/IISUrlRewrite/UrlRewriteFileParser.cs [hunk 1]
```diff
@@ -114,7 +114,7 @@ private void ParseConditions(XElement conditions, UrlRewriteRuleBuilder builder,
             var grouping = ParseEnum(conditions, RewriteTags.LogicalGrouping, LogicalGrouping.MatchAll);
             var trackAllCaptures = ParseBool(conditions, RewriteTags.TrackAllCaptures, defaultValue: false);
             var adds = conditions.Elements(RewriteTags.Add);
-            if (adds == null || adds.Count() == 0)
+            if (adds == null || !adds.Any())
```
Comment (davidfowl) [inline src/Middleware/Rewrite/src/Internal/IISUrlRewrite/UrlRewriteFileParser.cs]: Can this actually return null?

**Comment by davidfowl** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=237595

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2303

**Comment by davidfowl** [comment]
Thanks!

**Pull request merged**